content/id/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -0,0 +1,366 @@
+
+```shell
+kubectl scale deployment my-nginx --replicas=0; kubectl scale deployment my-nginx --replicas=2;
+
+kubectl get pods -l run=my-nginx -o wide
+```
+```
+NAME                        READY     STATUS    RESTARTS   AGE     IP            NODE
+my-nginx-3800858182-e9ihh   1/1       Running   0          5s      10.244.2.7    kubernetes-minion-ljyd
+my-nginx-3800858182-j4rm4   1/1       Running   0          5s      10.244.3.8    kubernetes-minion-905m
+```
+
+Kamu melihat *Pod* memiliki nama yang berbeda, karena mereka dihapus dan dibuat ulang.
+
+```shell
+kubectl exec my-nginx-3800858182-e9ihh -- printenv | grep SERVICE
+```
+```
+KUBERNETES_SERVICE_PORT=443
+MY_NGINX_SERVICE_HOST=10.0.162.149
+KUBERNETES_SERVICE_HOST=10.0.0.1
+MY_NGINX_SERVICE_PORT=80
+KUBERNETES_SERVICE_PORT_HTTPS=443
+```
+
+### DNS
+
+Kubernetes menawarkan sebuah layanan *DNS* kluster tambahan yang secara otomatis memberikan sebuah nama *dns* pada *Service*. Kamu dapat mengecek jika *DNS* berjalan didalam kluster Kubernetes:
+
+```shell
+kubectl get services kube-dns --namespace=kube-system
+```
+```
+NAME       TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)         AGE
+kube-dns   ClusterIP   10.0.0.10    <none>        53/UDP,53/TCP   8m
+```
+
+Jika *DNS* belum berjalan, kamu dapat [mengaktifkannya](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/kube-dns/README.md#how-do-i-configure-it).
+
+Sisa panduan ini mengansumsikan kamu mempunyai *Service* dengan *IP* (my-nginx), dan sebuah server *DNS*  yang memberikan nama kedalam *IP* tersebut (CoreDNS kluster),jadi kamu dapat berkomunikasi dengan *Service* dari *Pod* lain didalam kluster menggunakan metode standar (contohnya *gethostbyname*). Jalankan aplikasi *curl* lain untuk melakukan pengujian ini:
+
+```shell
+kubectl run curl --image=radial/busyboxplus:curl -i --tty
+```
+```
+Waiting for pod default/curl-131556218-9fnch to be running, status is Pending, pod ready: false
+Hit enter for command prompt
+```
+
+Lalu tekan *enter* dan jalankan `nslookup my-nginx`:
+
+```shell
+[ root@curl-131556218-9fnch:/ ]$ nslookup my-nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10
+
+Name:      my-nginx
+Address 1: 10.0.162.149
+```
+
+## Mengamankan Service
+
+HIngga sekarang kita hanya mengakses *nginx* server dari dalam kluster. Sebelum mengekspos *Service* ke internet, kamu harus memastikan bahwa kanal komunikasi aman. Untuk melakukan hal tersebut, kamu membutuhkan:
+
+* Self signed certificates* untuk *https* (kecuali jika kamu sudah mempunyai *identity certificate*)
+* Sebuah server *nginx* yang terkonfigurasi untuk menggunakan sertifikat tersebut
```

Comment:
I think *certificate* is more consistent than sertifikat (with the translated pages) and `self-signed certificate`

same with the rest